### PR TITLE
Conda recipe for Fourier-Bessel Particle-In-Cell code (FBPIC).

### PR DESCRIPTION
**Pull request by slacalle**

### CODE REVIEW

File: recipes/fbpic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - pyculib
+    - python-dateutil
+    - setuptools
+    - funcsigs
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - numba
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/fbpic/meta.yaml]: Do you really needs ``pytest`` to run that?
Or is it necessary just for testing?

File: recipes/fbpic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  sha256: '{{ hash }}'
+
+build:
+  skip: True    # [win]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/fbpic/meta.yaml]: Do you need ``pytest`` to build the library?

File: recipes/fbpic/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    - scipy
+    - h5py
+    - mkl
+    - mpi4py
+    - pyculib
+    - python-dateutil
+    - funcsigs
+
+test:
```
Comment (marcelotrevisani) [inline recipes/fbpic/meta.yaml]: Please, also add inside of this ``test`` section
```
requires:
    - nose
```

Comment (slacalle) [reply recipes/fbpic/meta.yaml]: Thank you for the advise @marcelotrevisani , much obliged. Have carried out the changes.

**Comment by slacalle** [comment]
Could this be merged, please?

**Comment by slacalle** [comment]
@conda-forge/staged-recipes  Could this be merged, please?

File: recipes/fbpic/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    - mkl
+    - mpi4py
+    - pip
+    - python-dateutil
+    - funcsigs
+
+  host:
+    - cudatoolkit
+    - python
+    - pytest
```
Comment (CJ-Wright) [inline recipes/fbpic/meta.yaml]: Is this a host dependency?

File: recipes/fbpic/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  host:
+    - cudatoolkit
+    - python
+    - numba
+    - scipy
+    - h5py
+    - mkl
+    - mpi4py
+    - pyculib
+    - pip
```
[the remark this answers is not in the document]
Comment (isuruf) [reply recipes/fbpic/meta.yaml]: why? `pip` should be in host, not build.

[the remark this answers is not in the document]
Comment (slacalle) [reply recipes/fbpic/meta.yaml]: Right. Just realized what you meant - Correct, removed dependency for pip in build section

File: recipes/fbpic/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/fbpic/meta.yaml]: Yeah this doesn't seem like a build dependency

File: recipes/fbpic/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  build:
+    - python
+    - pytest
+    - numba
+    - scipy
+    - h5py
+    - mkl
+    - mpi4py
+    - python-dateutil
+    - funcsigs
```
Comment (scopatz) [inline recipes/fbpic/meta.yaml]: Actually, most of these seems like host deps, not build deps

**Pull request closed without merging**